### ISSUE TITLE
Zenburn Theme Background problems

### ISSUE THREAD

**Issue by thaxyz** [codemirror/CodeMirror#6180]
Zenburn Theme: background ccss doesn't apply after default css is loaded.

.cm-s-zenburn {...}
Modify with
.cm-s-zenburn, .cm-s-zenburn.CodeMirror {...}

[zenburn.zip](https://github.com/codemirror/CodeMirror/files/4346106/zenburn.zip)

**Comment by marijnh** [comment]
Could you be more clear? How is the zenburn theme different from the others? At a glance, they seem to use the same level of specificity.

**Comment by thaxyz** [comment]
In my CM interface, I  have made a <Select> which allows to choose the theme. Thus, it changes without reloading the editor (using .setOption("theme", <theme>).
At page load, I load the "default" theme. 
When I change theme, the Zenburn options of class ".cm-s-zenburn" are not applied (most others work fine).
Thus, it misses global background and foreground colors.
By checking in other themes, I saw that they did not use ".cm-s-<theme>" for the global BG/FG but the class  definition " .cm-s-<theme>.CodeMirror".
After making the mod in the Zenburn.css, it works fine.

Using CM v 5.52.0

I did not check for every theme, as I wanted to use Zenburn.

:-)
